src/include/kdbprivate.h

```diff
@@ -281,6 +281,8 @@ typedef struct _KDB KDB;
  * @see kdbOpen() and kdbClose() for external use
  * @ingroup backend
  */
+typedef enum{PREROLLBACK = 0, POSTROLLBACK, PREGETSTORAGE, POSTGETSTORAGE, PRESETSTORAGE, PRECOMMIT, POSTCOMMIT, NR_GLOBAL_PLUGINS}Globalpluginpositions;
```

Comment:
is this also used for non-global plugins.

The enum should not be here, otherwise the comment above will refer to it.